tests/network/test_remove_intercept.py

```diff
@@ -21,7 +21,12 @@
 
 @pytest.mark.asyncio
 async def test_remove_intercept_no_such_intercept(websocket):
-    with pytest.raises(Exception) as exception_info:
+    with pytest.raises(
```

Comment:
Not related to the PR: This should be `invalid argument`, as `params` are not provided.

Reply:
They are provided:

```
        await execute_command(
            websocket, {
                "method": "network.removeIntercept",
                "params": {
                    "intercept": "00000000-0000-0000-0000-000000000000",
                },
            })
```

Reply:
Just expand the diff